### DISCUSSION TITLE
please add disney junior latin america and spain

### DISCUSSION THREAD

**Discussion by ahopgh** [iptv-org/community#578] [Q&A]
i am looking for links

**Comment by Mavijak** [comment]
i have a stream for Disney Jr LA And Spain: 
http://168.181.122.234:8000/play/a037/index.m3u8
http://85.49.100.235:5113/byname/DISNEY-JR

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/new?template=4_channel-search.yml
